requirements.in

```diff
@@ -35,7 +35,7 @@ pytest-dotenv
 pytest-env==0.6.2
 pytest==6.2.5
 python-dotenv
-sentry-sdk
+sentry-sdk==1.11.1
```

Comment:
It hasn't been updated when running `pip-compile` and has remained stuck at version `1.5.4` for a while. Should I be running another command to update libraries?

Reply:
Running `pip-compile -U` should update the unpinned packages to the latest version which is compatible with the other requirements.

Reply:
Running `pip-compile -U` should update the unpinned packages to the latest version which is compatible with the other requirements.

Reply:
I removed the pin, however I couldn't run `pip-compile -U` due to `click==7.0` and `Django>=3.2,<4.0` pins as they have conflicts from other packages where they are dependencies. I will leave this for now until we are able to upgrade to Django>4.0 to resolve some of these issues.

Reply:
I've had a look at this myself. I'd remove the pinned version of `click` as the application uses `django-click` rather than `click` itself directly.

Then add a pin on `django-admin-sortable2==1.0.4` because version 2 requires django 4.

After you've done that - all should be well with the package upgrades.

Reply:
Ok, I have upgraded all the requirements except the pinned ones for now.